documentation/operators/src/nodes/maintenance.md

```diff
@@ -399,6 +298,29 @@ WantedBy=multi-user.target
 ```
 * Put the above file onto your system at `/etc/systemd/system/nym-network-requester.service` and follow the [next steps](maintenance.md#following-steps-for-nym-nodes-running-as-systemd-service).
 
+##### For Nymvisor
+> Since you're running your node via a Nymvisor instance, as well as creating a Nymvisor `.service` file, you will also want to **stop any previous node automation process you already have running**.
```

Comment:
I would consider to drop example How to stop the node automation service or linked it, to prevent operators try and error. But maybe this is just me seeing how little people generally want to seek for solution and self educate (as we have it clearly written below).

Reply:
This is already such a big doc I'd prefer to leave it and lets see. Since we don;t know how people are running their nodes

Reply:
Makes sense.